handlers/mesh_ops_handlers.go

```diff
@@ -108,50 +106,46 @@ func (h *Handler) addAdapter(ctx context.Context, meshAdapters []*models.Adapter
 	}
 
 	if alreadyConfigured {
-		logrus.Debugf("Adapter already configured...")
+		h.log.Debug("Adapter already configured...")
 		return meshAdapters, nil
 	}
 	if prefObj.K8SConfig == nil {
-		err := fmt.Errorf("k8s config is nil")
-		logrus.Error(err)
-		return nil, err
+		h.log.Error(ErrInvalidK8SConfig)
+		return nil, ErrInvalidK8SConfig
 	}
 
 	kubeclient, err := mesherykube.New(prefObj.K8SConfig.Config)
 	if err != nil {
-		err = fmt.Errorf("unable to create Kubernetes client")
-		logrus.Error(err)
-		return nil, err
+		h.log.Error(ErrNilClient)
+		return nil, ErrNilClient
 	}
 	*h.kubeclient = *kubeclient
 	provider.SetKubeClient(h.kubeclient)
 
 	mClient, err := meshes.CreateClient(ctx, prefObj.K8SConfig.Config, prefObj.K8SConfig.ContextName, meshLocationURL)
 	if err != nil || prefObj.K8SConfig == nil {
-		err = errors.Wrapf(err, "Error creating a mesh client.")
-		logrus.Error(err)
-		// http.Error(w, "Unable to connect to the Mesh adapter using the given config, please try again", http.StatusInternalServerError)
-		return meshAdapters, err
+		h.log.Error(ErrMeshClient)
+		// http.Error(w, ErrMeshClient.Error(), http.StatusInternalServerError)
+		return meshAdapters, ErrMeshClient
 	}
-	logrus.Debugf("created client for adapter: %s", meshLocationURL)
+	h.log.Debug("created client for adapter: %s", meshLocationURL)
```

Comment:
`h.log.Debug` uses `logrus.Log` internally, and not `logrus.Logf`.

you might want to rewrite it as
```golang
h.log.Debug("created client for adapter: ", meshLocationURL)
```

Reply:
okay so all the h.log.Debug should be changed right?